app/helpers/profile_helper.rb

```diff
@@ -15,4 +15,13 @@ def social_authentication_links_for(user)
 
     { github: urls["GitHub"], twitter: urls["Twitter"], facebook: urls["Facebook"] }.compact
   end
+
+  def character_count_denominator(field_type)
+    case field_type
```

Comment:
At first I had a ternary here, but then thought it'd be better to have the function accept more than just `text_field` and `text_area` arguments. Feels more stable this way. Open to thoughts.